packages/frontend/public/hamh-logo.svg

```diff
@@ -0,0 +1,69 @@
320vXBVYC&#10;zaiPe5t14strQ2DTfkoBcDklZYEL3CLbbpifRbfrTtgS8NlfUheJb1MOBtUJHwp9Kb5VPyV1gLdw&#10;T+xrnRRU29HCtFJktUj4FPi5J+hVkSWh5tjXvnHLcrwwSvOybpYSm/NMADzPKAi/kNKcqjLPj4+W&#10;52VV6xuMLsY2BsB/J/fiXJf6Q1EsNk15CKIlSw+Aj4D8aXcW5s2I5BeR9rjWBuCu+YNpDchflum4&#10;PBsjUK/Umfc8ocABrJjWgPwt2nvQB//x8/D3qUs6gKfNalINOBsY+vG9B44JwB0fgMeR8KsaSn+T&#10;aldeaAYAPxZ/8wJ4Kgg8RC8w9OxjLIoBwB+L7/wAHnvBfS1b/+agfYpBMAD4L8tBIG87uKa6zBVB&#10;KdrSUhgArFgOAjkhZ0me/vq+9YQ9LABTjWo5neCcEc7WAwMwTfcgQ39JjEIPDsA074V3gnNG9jaP&#10;AIDqCZ4J9H1H/HkEAB63x6lAT5bRnYYSLkcAwFT3ws4HRtrtXh0HANPUSjETgR715Q6MBIDpCJkI&#10;6t50zNEAMK0U/3BQ9WdejAfA4yl5g6yfJgtvRABMM0JdDIycJtY8KgBmiOndP1JF18YFwPyF1gcM&#10;Ov1jA4DXByj1m74kAHVE2aAIh0BO9+eskCWJii0wWCcuVUpogrEjKqhybZx9wfTHGCOjRnOkSQ63&#10;Un4ClUcjP6kZdx7soeHmQJEYbHPvCWuKK1lOxO6S5ImN62l/53Q4feZK/2Uca8/jhOBLDqjuezvB&#10;jiuEaCS92RScvkjO7Ihl3jtC9zwEsr4FwE45Y7Tc6SH1r77sn4Tj431jzOdOnufPj1GynlbazI3U&#10;ey7hWDF/iB4jQajcCbjh92Rxz3czhG0GSoaUvu/ZEbI1NO8eXBpKUBYnRewcd64GW6aJuna7r47j&#10;hCOxcuQOnTmALkNjjW4PYIiUoYMFwOhesCKGI8BWwjkLD0DPmQ0+CHRNin5EAN19QIP22LRrVonx&#10;fK6YebKdfosUOAP+T4qvCReA0fWrOaB4kRF2hZ4wvY2omdLnAGszUNjiDpjCABiFhlbxJOi+PtqM&#10;EUAZ9hyNAUthofYcgbPRrQJ6fzEhi3o3pPR6gawQ6x4dDoCPGYuEyZu2zTRuYLYCm2IALAu6G+PU&#10;s9c9XRkdlHGwwBj8tBeiYkpHG+0tM/9oDA9AT+jvg7lUGxgjov6gFVQDA1AyUO2YDMUPhjoVcAEw&#10;SmDNEI+mCzzAvukelaEAVBH0SihVnGQH/CifZ5QjMpQy3I+nmAYrhgvWHKERVgCbbMdSMcdVEKfA&#10;9ytsrEsFsKxYq0b1jgGdsSKXlyvyABCy1VGSOHLWQkRqUEgC0GQcxTF63aMx+7dtlnx9OIDjzuSw&#10;PufghqsOk5Y0ggEYVcBZFCNCXwOuEUCnAhgAPeIukeR1A455v78NSmEAzhjFgdTO3fvZ4/8LFmgq&#10;AADIQUsfsTZ0p3u4IiEGLY7uQUEHsKz2kMFvx1VE2CgFXVnEOXHxeID0PnVHPRUsaAc/5O9vdwXZ&#10;rdP5mE5KnjpPoPnHCitEAE0O64GX+i+kLZ3d1TJSOOBpYluWoDFoJycDCQCoUKbqxOdOMWqHU0Ah&#10;9DPn5ZMZZJdA9yhNLwCjjizQFPzbXZvDc4ZIu4D9a2N02BHULxReAOcYNPm8P5rXhDEbkv9a1r9y&#10;VCFkKrB7p4JuAGfg4M8aCs+GZkCPwldpVkoBqtZo99zq7gQAO/Q6H3O2CHIsHRoQ+RBY3RxciC9C&#10;y5gBpCCvzKcqCg+E/0rslQ1hjvM+/IbGKYbMS64MAH5OH+IlXrE+udQHqBpQok4CACdrAPvaGBoR&#10;aTtCt7xqNRgA+759VDeEHzQkzUqEvCjC+WmTuuMAEJBKqCwJRzuPtDgfCYB10haF7n0boQBUrcPF&#10;Q0gg1UjhkUP75K6RdzOFvx0WgNNZ8RT6siTh7+26YnOFNyAAe9/t4iRs7NwTDFjYvVlL3IEAqGHZ&#10;4+MmnAYcEraIyY34OBVYQwDw+pPVS2CiBBuApz8UbGUD2MYUhZMqFeYRCJkDikrXoUUAAJvO3wIF&#10;0H70UKnSy87kWub4APyCLlcXCiDgALBaEh0X7ADaT4PeA63DUSoA4q5gd2T3B7QMLTuhr5omcwg8&#10;j4PPF3qdA4c/4Gk/fz20qJ2twwB48h071zsVPo/QR5/grgBF3kqgX5h5GbyKXr4LYPhl72RF4xV+&#10;DQlp0CJRhax9wMf2Xnqe6h0oBitVXODiGrSiEzT6ngFzBAhb4V/QkPf5KXnPpnuUhTYyFIflCmwJ&#10;zmHIh/9lPd5jRoaehhbLLZ0I5zjsMdQkom6vyNr6hg91iKgghwiKiQRw1oCJUjnIJTZ+AKT3yCOY&#10;U1QNb7UHkOKKxBSJ1jRprxD64ITo9zWKlhp4BSAw4qaCq/gLf2BE+bQ9J98dUrxPseaT6CKtEl5Y&#10;+XifyiF26Y/B0aAQ3zopT8xcJxcG5P/4cJ1oIqNku5w3doz3GWYdHq53CRJOIucJD1mPDL2VWlE7&#10;yk83rz3FklauXtorS6vXOFZnUZnda6xZWql6eQAeT+qXx/eIYZHf50E3V+S1SiKA1Wpz3Knkk8Dl&#10;NPCUaHKS2SapAC7pxp2FpSotkPxWhWQAjxI7BSqVspo4gLGZjMcGxw3g73nr/4floaUp2d1i5pPA&#10;YrH4Nmf93xaLeY+By4uT3+bdARZzngVent39Plf9v9/enulmaP32+vr3meuf41KwWXywmQ2Dfxaf&#10;bb2Zza//Y0Gyu39N3u4WX/ZlX/ZlX/ZlL/Z/2531pK2xQ+kAAAAASUVORK5CYII=&#10;"
```

Comment:
Is there a png image embedded? This does not scale well...

Reply:
ohh shit I looked for the symbol on the internet now I am modifying it again